pkg/server/server.go

```diff
@@ -94,12 +94,20 @@ func WithTransport(t gateway.Transport) OpenFGAServiceV1Option {
 	}
 }
 
+// WithResolveNodeLimit sets a limit on the number of recursive calls that one Check or ListObjects call will allow.
+// Thinking of a request as a tree of evaluations, this option controls
+// how many levels we will evaluate before throwing an error that the authorization model is too complex.
 func WithResolveNodeLimit(limit uint32) OpenFGAServiceV1Option {
 	return func(s *Server) {
 		s.resolveNodeLimit = limit
 	}
 }
 
+// WithResolveNodeBreadthLimit sets a limit on the number of goroutines that can be created
+// when evaluating a subtree of a Check or ListObjects call.
+// Thinking of a Check request as a tree of evaluations, this option controls,
+// on a given level of the tree, the maximum number of nodes that can be evaluated concurrently (the breadth).
+// If your authorization models are very complex, you should set this option to be a low number (e.g. 1000)
```

Comment:
> If your authorization models are very complex, you should set this option to be a low number (e.g. 1000)

Should this number related to the server/database capability? The way it is phrased suggest there is a need to tweak this value as you have more/less authorization models. In some case, the server operator has no control on the model complexity.

Reply:
I'm not exactly sure what is the relation between goroutines that you can create and CPU or memory of the OpenFGA server, that's why i left it ambiguous for now.

Reply:
In this case, probably just remove the last line `If your authorization models are very complex, you should set this option to be a low number (e.g. 1000)` as it may cause more confusion than intended

Reply:
It's not just models, it's the data (e.g. the tuples). This doc needs to account for that as well.

> If your authorization models have a large number of relationship rewrites for a single relation or if you have a large degree of breadth in your relationship tuples, then you should set this option to bound the concurrency appropriately.

Reply:
updated